docs/detections/rules-ui-create.asciidoc

```diff
@@ -263,35 +263,100 @@ For example, if a rule has an interval of 5 minutes, no additional look-back tim
 [[create-esql-rule]]
 === Create an ES|QL rule
 
-IMPORTANT: This is a placeholder for future documentation. The following content is incomplete.
+preview::[]
 
-. Go to *Rules* -> *Detection rules (SIEM)* -> *Create new rule*. The *Create new rule* page displays.
-. To create a rule that uses ES|QL, select **ES|QL**, 
-then write a query. There are two types of ES|QL queries:
+Use {ref}/esql.html[ES|QL] to query your source events and aggregate event data. Query results are returned in a table with rows and columns. Each row becomes an alert.
+
+To create an ES|QL rule:
 
-.. **Aggregating query**: This is a query that uses the `STATS...BY` grouping commands. Query results cannot be matched with a particular document in Elasticsearch. For example:
+. Go to *Rules* -> *Detection rules (SIEM)* -> *Create new rule*. The *Create new rule* page displays.
+. Select **ES|QL**, then write a <<esql-rule-query-types,query>>. 
```

Comment:
```suggestion
. Select **ES|QL**, then write a <<esql-rule-query-types,query>>. 
```
Not sure the link is necessary here since it only jumps a few lines. Maybe it's just me but following it was a little disorienting

Reply:
Yeah, I've gone back and forth on this too. Other than implying that there's more to learn, it's not adding much value 😅